public/apps/account/tenant-switch-panel.tsx

```diff
@@ -91,7 +92,12 @@ export function TenantSwitchPanel(props: TenantSwitchPanelProps) {
         setUsername(currentUserName);
 
         // @ts-ignore
```

Comment:
Could you remove the `@ts-ignore` statement?

Reply:
Done